test/matmul.jl

```diff
@@ -7,8 +7,17 @@ include("../configs/configs.jl")
 
 @testset "Matrix multiplication" begin
     @testcase "$( cf.name )" for cf in get_configs()
-        c_h, a, b, c, d = generate_inputs(cf)
-        run_gemm(cf, a, b, c, d)
-        @test verify(cf, c_h, d)
+        try
+            c_h, a, b, c, d = generate_inputs(cf)
+            run_gemm(cf, a, b, c, d)
+            @test verify(cf, c_h, d)
+        catch err
+            # Count tests with config errors as "broken".
+            if isa(err, GemmKernels.ConfigError)
+                @test true skip=true
```

Comment:
@maleadt For now, I've marked unsupported configurations as "Broken". The term is not really appropriate, as broken tests should be tests that ought to pass, but do not currently. We could also just mark these as "pass", but I quite like that they are reported separately so we can easily see how many configurations are skipped due to `ConfigError`s. WDYT?

Reply:
Ideally we hard-code whether a configuration is unsupported, i.e., which configuration we know is supposed to throw a ConfigError. That way the `broken` would work as intended, resulting in test failures when it anything starts failing/passing without updating the tests. However, I guess that would be hard to do, given how we now generate configurations using multiple loops. Maybe we could maintain a separate list of known-broken configurations? I guess it would also depend on the device though (because of the shmem limitation), so maybe that's not feasible either...